src/ripple/app/ledger/LedgerTiming.cpp

```diff
@@ -20,31 +20,26 @@
 #include <BeastConfig.h>
 #include <ripple/app/ledger/LedgerTiming.h>
 #include <ripple/basics/Log.h>
+#include <algorithm>
+#include <iterator>
 
 namespace ripple {
 
-// VFALCO Should rename ContinuousLedgerTiming to LedgerTiming
-
-// NOTE: First and last times must be repeated
-int ContinuousLedgerTiming::LedgerTimeResolution[] = { 10, 10, 20, 30, 60, 90, 120, 120 };
-
-// Called when a ledger is open and no close is in progress -- when a transaction is received and no close
-// is in process, or when a close completes. Returns the number of seconds the ledger should be be open.
-bool ContinuousLedgerTiming::shouldClose (
+bool shouldCloseLedger (
     bool anyTransactions,
-    int previousProposers,      // proposers in the last closing
-    int proposersClosed,        // proposers who have currently closed this ledgers
-    int proposersValidated,     // proposers who have validated the last closed ledger
-    int previousMSeconds,       // milliseconds the previous ledger took to reach consensus
-    int currentMSeconds,        // milliseconds since the previous ledger closed
-    int openMSeconds,           // milliseconds since the previous LCL was computed
-    int idleInterval)           // network's desired idle interval
+    int previousProposers,
```

Comment:
1. Why did you get rid of these comments?
2. Now you're cleaning this up anyway, shouldn't these all be `uint`?